vex-input.json

```diff
@@ -0,0 +1,468 @@
+[
+  {
+    "ids": [
+      "CVE-2022-33980"
+    ],
+    "versions": "< 9.1",
+    "jars": [
+      "commons-configuration2-2.7.jar"
+    ],
+    "analysis": {
+      "state": "not_affected",
+      "detail": "Solr uses commons-configuration2 for \"hadoop-auth\" only (for Kerberos). It is only used for loading Hadoop configuration files that would only ever be provided by trusted administrators, not externally (untrusted)."
+    }
+  },
```

Comment:
Can we use some tooling/editor or at least a linter when adding entries to this json file, so we're sure it is parsable? Will probably get a Pelican build failure if something is wrong?